tests/common.bash

```diff
@@ -547,6 +547,24 @@ EOF
 	sudo systemctl enable --now crio
 }
 
+function install_docker() {
```

Comment:
Now that https://github.com/kata-containers/kata-containers/issues/8357 has landed, could we use `kata-manager` instead here?

Reply:
No, it doesn't make sense to change to using kata-manager as part of this series that's just re-arranging things.

We need, though, an issue to start using kata-manager to install docker / nerdctl etc, and then replace the containerd / ctr / docker / nerctl deployment throughout our CI.

Doing this here will just make things not consistent, and is out of the scope of this PR.